api/v1beta1/azureclusteridentity_types.go

```diff
@@ -77,6 +77,11 @@ type AzureClusterIdentityStatus struct {
 }
 
 // +kubebuilder:object:root=true
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels.cluster\\.x-k8s\\.io/cluster-name",description="Cluster to which this AzureClusterIdentity belongs"
+// +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
+// +kubebuilder:printcolumn:name="Severity",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].severity"
+// +kubebuilder:printcolumn:name="Reason",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].reason"
+// +kubebuilder:printcolumn:name="Message",type="string",priority=1,JSONPath=".status.conditions[?(@.type=='Ready')].message"
```

Comment:
Are these fields meaningful for an AzureClusterIdentity? I don't see where we'd ever set `status.conditions` at all for one.

Reply:
So should I remove all fields?

Reply:
Yes, I think we can remove all of these. The `spec.type` one currently defined on AzureManagedCluster could still be useful though.